pydantic/config.py

```diff
@@ -62,10 +62,10 @@ class ConfigDict(TypedDict, total=False):
         getter_dict: Type[GetterDict]
         alias_generator: Optional[Callable[[str], str]]
         keep_untouched: Tuple[type, ...]
-        schema_extra: Union[Dict[str, Any], 'SchemaExtraCallable']
-        json_loads: Callable[[str], Any]
-        json_dumps: Callable[..., str]
-        json_encoders: Dict[Type[Any], AnyCallable]
+        schema_extra: Union[Dict[str, object], 'SchemaExtraCallable']
```

Comment:
are these really all required?

I would rather keep the change as small as possible while fixing the original issue.

Reply:
yeah they are, changing any one of them back to `Any` causes the mypy error to come back. i made sure to only change types that were absolutely necessary to make the error go away.